features/dd-sdk-android-session-replay/src/main/kotlin/com/datadog/android/sessionreplay/internal/SessionReplayFeature.kt

```diff
@@ -92,10 +99,12 @@ internal class SessionReplayFeature(
         }
 
         this.appContext = appContext
-
         sdkCore.setEventReceiver(SESSION_REPLAY_FEATURE_NAME, this)
+        if (RESOURCES_ENDPOINT_ENABLED) {
```

Comment:
it is always `false`, so I guess I guess for now it is a feature flag?

Reply:
yes - it's a ff that we'll remove in a later pr